pkg/virt-handler/vm.go

```diff
@@ -907,7 +907,7 @@ func (d *VirtualMachineController) updateLiveMigrationConditions(vmi *v1.Virtual
 
 	evictable := migrations.VMIMigratableOnEviction(d.clusterConfig, vmi)
 	if evictable && liveMigrationCondition.Status == k8sv1.ConditionFalse {
-		d.recorder.Event(vmi, k8sv1.EventTypeWarning, v1.Migrated.String(), "EvictionStrategy is set but vmi is not migratable")
+		d.recorder.Event(vmi, k8sv1.EventTypeWarning, v1.Migrated.String(), fmt.Sprintf("EvictionStrategy is set but vmi is not migratable (%s)", liveMigrationCondition.Message))
```

Comment:
I think more common is to format the string as follows: `"EvictionStrategy is set but vmi is not migratable` + `:` or `,` ...

Reply:
Fixed